### PR TITLE
Added "Procedures" in Patient Dashboard.

### LINKED ISSUE #4871: View procedures in Patient Dashboard
**Is your feature request related to a problem? Please describe.**
We are recording procedures suggested for a patient in the consultation form (https://care.coronasafe.in/facility/e6b40cf4-4196-4e7d-83f0-b0bf5c38458a/patient/b80bbd73-8d8b-47ba-8a71-12a2bbfd65b2/consultation) However, we are not viewing the information anywhere.

**Describe the solution you'd like**
Add a card in the Patient Dashboard below "General Instructions" to show additional details about each of the procedures suggested. Keep the design consistent with the "prescription" and "General Instructions" cards.

### PR DESCRIPTION
**Pull request by jeromtom**
## Proposed Changes

- Fixes #4871
- Add a card in the Patient Dashboard below "General Instructions" to show additional details about each of the procedures suggested keeping the design consistent with the "Body Details" card.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by jeromtom** [comment]
Cypress Tests are failing as usual. Let me know if any other change is required.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@jeromtom 

1. repetitive is not being shown
2. table contents are clipped, not scrollable even.

image


image


Patient Consultation: https://deploy-preview-5034--care-net.netlify.app/facility/7ab27974-32c8-4e36-bdf9-dbf102524718/patient/36cdd810-c3e1-45dd-993c-2877a46e35de/consultation/48f289bb-8ae2-46d8-b6b1-4840fa6f2385

**Comment by jeromtom** [comment]
@rithviknishad , understood will work on it. Do we have to show the "Repetitive" field? How about showing either "Time" or "Frequency" depending on if repetitive field was selected.

**Comment by rithviknishad** [comment]
@jeromtom I agree with that.

cc: @aparnacoronasafe

File: src/Components/Common/components/ResponsiveProcedureTables.tsx
Comment (rithviknishad) [inline src/Components/Common/components/ResponsiveProcedureTables.tsx:1]: @jeromtom I'm still not able to do a horizontal scroll, nor contents visible.

How about using `col-span-2 scroll-x-auto`?

image

File: src/Components/Common/components/ResponsiveProcedureTables.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+    window.addEventListener("resize", handleWindowResize);
+
+    return () => {
+      window.removeEventListener("resize", handleWindowResize);
+    };
+  }, []);
+  return (
+    <>
+      {windowSize.innerWidth > 1024 ? (
```
Comment (rithviknishad) [inline src/Components/Common/components/ResponsiveProcedureTables.tsx:29]: I don't see a reason to have this much complication. Wouldn't tailwind's modifiers be enough?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Pull request closed without merging**